cookbooks/bcpc-hadoop/recipes/datanode.rb

```diff
@@ -102,6 +128,7 @@
   shell "/bin/bash"
   home "/usr/lib/hcatalog"
   supports :manage_home => false
+  not_if { user_exists? "hcat" }
```

Comment:
This will not work as `user_exists` method is part of Kerberos pull request #117